qiskit/visualization/circuit/matplotlib.py

```diff
@@ -1058,6 +1059,14 @@ def _draw_ops(
                     ]
                     self._condition(node, node_data, wire_map, outer_circuit, cond_xy, glob_data)
 
+                # AnnotatedOperation with ControlModifier
+                mod_control = None
+                if getattr(op, "modifiers", None):
+                    for modifier in op.modifiers:
+                        if isinstance(modifier, ControlModifier):
+                            mod_control = modifier
+                            break
+
```

Comment:
Please see the comment in `text.py`.

Reply:
Got it. I understand that displays are sometimes 'best efforts'. Changes done in https://github.com/Qiskit/qiskit/pull/11202/commits/0c79f189d26db07a1bcdf5dda4dca4f58b071680.